dashboard/lib/tasks/seed.rake

```diff
@@ -333,15 +354,19 @@ namespace :seed do
     sh('mysqldump -u root -B dashboard_test > db/ui_test_data.sql')
   end
 
-  task :cache_ui_test_data do
-  end
+  FULL_SEED_TASKS = [:videos, :concepts, :scripts, :callouts, :school_districts, :schools, :secret_words, :secret_pictures, :courses, :ap_school_codes, :ap_cs_offerings, :ib_school_codes, :ib_cs_offerings, :state_cs_offerings, :donors, :donor_schools, :foorms].freeze
+  UI_TEST_SEED_TASKS = [:videos, :concepts, :scripts_ui_tests, :courses_ui_tests, :callouts, :school_districts, :schools, :secret_words, :secret_pictures, :donors, :donor_schools].freeze
```

Comment:
I found these a bit hard to compare visually, because the order is different. If it's not too annoying, could `:courses` be moved to just after `:scripts` in the first list so that the orders match?

Reply:
I think I copy / pasted these directly, I'm ok with making an improvement though. An even better improvement might be to refactor the shared tasks into a constant, and then build `FULL_SEED_TASKS` and `UI_TEST_SEED_TASKS` by concatenating the ones that differ. I'll try that out.

Reply:
Never mind, it gets too complicated with `seed:incremental` in the picture too. I assume order also matters so I don't want to mess with that too much. I'll just make the change ou suggested.